### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ps0305**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,10 +80,10 @@ react-native run-ios
   * in case of handler's failure, `failure` prop (if defined) will be run.
   * Combining `onEnter`, `onExit`, `success`, and `failure` makes patterns like authentication, data validation, and conditional transitions simple and intuitive.
 * [MobX](https://mobx.js.org/)-friendly: all scenes are wrapped with `observer`. You may subscribe to `navigationStore` (`Actions` in v3) and observe current navigation state. Not applicable to Redux.
-* Flexible nav bar customization not currently allowed by React Navigation:
+* Flexible Nav bar customization ,currently not allowed by React Navigation:
 https://github.com/react-community/react-navigation/issues/779
```
Comment (aksonov) [inline README.md]: Is this comma correct?

**Comment by aksonov** [comment]
Thanks!

File: README.md (hunk 1 above)
Comment (ps0305) [reply README.md]: nope,it got misplaced !

File: README.md [hunk 2]
```diff
@@ -78,10 +80,10 @@ react-native run-ios
   * in case of handler's failure, `failure` prop (if defined) will be run.
   * Combining `onEnter`, `onExit`, `success`, and `failure` makes patterns like authentication, data validation, and conditional transitions simple and intuitive.
 * [MobX](https://mobx.js.org/)-friendly: all scenes are wrapped with `observer`. You may subscribe to `navigationStore` (`Actions` in v3) and observe current navigation state. Not applicable to Redux.
-* Flexible nav bar customization not currently allowed by React Navigation:
+* Flexible Nav bar customization ,currently not allowed by React Navigation:
```
Comment (ps0305) [inline README.md]: Flexible Nav bar customization, currently not allowed by React Navigation:

Comment (aksonov) [reply README.md]: I see comma right before currently, so now it is "Flexible Nav bar customization ,currently not allowed by React Navigation:"

File: README.md (hunk 1 above)
Comment (aksonov) [reply README.md]: Could you fix it?

Comment (ps0305) [reply README.md]: sure

**Pull request merged**